lib/directory-view.coffee

```diff
@@ -35,6 +35,7 @@ class DirectoryView extends HTMLElement
     @directoryName.textContent = @directory.name
     @directoryName.dataset.name = @directory.name
     @directoryName.dataset.path = @directory.path
+    @directoryName.draggable = true
```

Comment:
Nesting of `.list-item` block differs in directory and file elements, so if I put draggable on it, I'll have to add some messy if-s to get initial and target paths. I can simply add `display: block` to `.name` block, so the draggable area would be rendered up to tree-view right border (attached screenshot). What do you think?

![preview2](https://cloud.githubusercontent.com/assets/3530199/8166445/8e233b28-13a0-11e5-9207-da5d42debdd9.png)